packages/core/src/amazonqDoc/types.ts

```diff
@@ -20,7 +20,7 @@ import {
     SessionState as FeatureDevSessionState,
     SessionStateAction as FeatureDevSessionStateAction,
     SessionStateInteraction as FeatureDevSessionStateInteraction,
-} from '../amazonqFeatureDev/types'
+} from '../amazonqFeatureCommon/types'
```

Comment:
Can we either use `src/amazonq` since theres already common q code there or `src/shared/amazonq` instead of creating a new convention